src/main/java/emissary/util/TimeUtil.java

```diff
@@ -250,24 +252,33 @@ public static ZonedDateTime getZonedDateFromISO8601(@Nullable final String dateS
      *
      */
     public static String convertHexDate(String hexDate) {
+        String regex = "^([0-9A-Fa-f]{8})([0-9A-Fa-f]{8})";
```

Comment:
If we're going to use a regex to parse the input string, we should consider adding a non-capturing group  that matches the "0x" prefix, rather than our current technique of stripping off the first 2 characters regardless of what they are (i.e,  `hexDate.substring(HEX_PREFIX_LEN)`). 
```suggestion
        String regex = "^(?:0x)([0-9A-Fa-f]{8})([0-9A-Fa-f]{8})";
```

We should also move the regex string and the compiled Pattern declarations out of this method and make them constants, since they're independent of the method input parameter.